vault/logical_system_test.go

```diff
@@ -2911,7 +2912,7 @@ func TestSystemBackend_rawReadWrite_Compressed(t *testing.T) {
 		)
 
 		// Read back and check gzip was applied by looking for prefix byte
-		req = logical.TestRequest(t, logical.ReadOperation, "raw/core/mounts")
+		req = logical.TestRequest(t, logical.ReadOperation, "raw/core/audit")
 		req.Data = map[string]interface{}{
 			"compressed": false,
 			"encoding":   "base64",
```

Comment:
@fatima2003 I forgot to mention yesterday, but remember how on last week's call we couldn't figure out how to read the certs entry with `sys/raw`? I had forgotten about `encoding=base64` as a parameter to the request. :D

Reply:
Ohh, I will be trying this out :)) Thank you!